dotnet/src/webdriver/ErrorResponse.cs

```diff
@@ -42,52 +44,41 @@ public ErrorResponse()
         /// </summary>
         /// <param name="responseValue">A <see cref="Dictionary{K, V}"/> containing names and values of
         /// the properties of this <see cref="ErrorResponse"/>.</param>
-        public ErrorResponse(Dictionary<string, object> responseValue)
+        public ErrorResponse(Dictionary<string, object?>? responseValue)
         {
             if (responseValue != null)
             {
-                if (responseValue.ContainsKey("message"))
+                if (responseValue.TryGetValue("message", out object? messageObj)
+                    && messageObj?.ToString() is string message)
```

Comment:
`&& messageObj is not null` instead of `&& messageObj?.ToString() is string message`?

Reply:
The problem is, untyped `object.ToString()` is annotated to return a `string?`. we need to handle that potentially null value too.

We can suppress it like before, with `messageObj.ToString()!`

Reply:
Ah, I like `objects` in selenium.. Looked through where and how this class used, and seems it makes sense to make fields nullable.

Reply:
I reworked the class to allow nullable fields. Let me know if it needs any more changes